src/_zkapauthorizer/config.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 PrivateStorage.io, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Helpers for reading values from the Tahoe-LAFS configuration.
+"""
```

Comment:
I wonder if we want to have separate modules for client and server config. Other than `empty_config` (which I guess is a test fixture), all the stuff here is client specific.

Reply:
Could be, yea.  Since this project started it has become clear that two different plugins may have made more sense than a single plugin with such a tightly coupled implementation.  We could probably try to keep the pieces as separate as possible until a more definitive decoupling happens.  I'll keep it in mind for future development.